CMakeLists.txt

```diff
@@ -144,6 +144,12 @@ if (INSTALL_HEADER_ONLY)
   set_target_properties(portblas PROPERTIES
     INTERFACE_INCLUDE_DIRECTORIES "${PORTBLAS_INCLUDE};$<INSTALL_INTERFACE:src>"
   )
+  set(non_intel_target "AMD_GPU" "NVIDIA_GPU")
+  if((${CMAKE_CXX_COMPILER_ID} STREQUAL "IntelLLVM") AND
```

Comment:
Stupid question: `IntelLLVM` mean the closed source or the open-source `intel/llvm`?

Reply:
Thank you for your review @hjabird 
`IntelLLVM` is the closed source. Cmake uses this name when referring to `icpx` or `clang++` from the oneAPI release, while the open source compiler is identified simply as `Clang`